src/main/java/com/example/demo/matcher/models/OrderbookItem.java

```diff
@@ -2,7 +2,6 @@
 
 import lombok.AllArgsConstructor;
 import lombok.Getter;
-import lombok.Setter;
```

Comment:
Good job getting rid of it here 👍

Reply:
IntelliJ has great tools for tidying code!